src/ItemResults/ItemResults.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import theme from 'src/styles/theme';
+import { ThemeProvider } from '@emotion/react';
+import { ItemStyleContainer } from './itemResultsStyles';
+
+export interface ItemResultsProps {
+    dataLength: number;
+    totalItems: number;
+}
+
+const ItemResults: React.FC<ItemResultsProps> = ({ dataLength, totalItems }) => {
+    const digitGroupSeparator = (number: number) => {
```

Comment:
Since this doesn't have any dependencies on the component itself, I think this would be better declared as a helper function outside the body of the component.  I don't feel too strongly, though; if we've done it this way elsewhere in Lakefront I'm OK to leave it.